app/controllers/test/ipp_controller.rb

```diff
@@ -21,17 +20,32 @@ def index
 
     def update
       enrollment_id = params['enrollment'].to_i
-      enrollment = InPersonEnrollment.find(enrollment_id)
+      enrollment = InPersonEnrollment.find_by(id: enrollment_id)
```

Comment:
Should this include the `user_id` filter so that I can't approve someone else's enrollment?